src/librepgp/stream-dump.cpp

```diff
@@ -1152,6 +1152,16 @@ stream_dump_literal(pgp_source_t *src, pgp_dest_t *dst)
     return ret;
 }
 
+static rnp_result_t
+stream_dump_marker(pgp_source_t &src, pgp_dest_t *dst)
```

Comment:
A bit strange that `src` is by reference, whereas `dst` is a pointer, is it not?

Reply:
Yeah, it's because inner function calls use `pgp_dest_t *`parameter. And once we'll change them to `pgp_dest_t &`, we'll need to change 4 lines of code again, while within current declaration we'll need to change single line :)

Reply:
I strongly believe newly-created functions should have nice signatures. You can call other functions with `&dst` for now.

Reply:
@rrrooommmaaa Ok, re-pushed. Any other comments?